RELEASE.md

```diff
@@ -1,3 +1,115 @@
+# Release 1.1.0
+
+## Major Features and Improvements
+* Added Java API support for Windows.
+* Added `tf.spectral` module. Moved existing FFT ops to `tf.spectral`.
```

Comment:
I reworded it to "Added `tf.spectral` module. Moved existing FFT ops to `tf.spectral` while keeping an alias in the old location (`tf.*`)."
since alias is the other way.